com.alizadicomviewer.AlizaMS.yaml

```
@@ -0,0 +1,158 @@
+    cleanup:
+      - /share
+
+  - name: gdcm
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DGDCM_BUILD_APPLICATIONS=OFF
+      - -DGDCM_BUILD_SHARED_LIBS=ON
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DGDCM_WRAP_PYTHON=ON
+      - -DPYTHON_EXECUTABLE=/usr/bin/python3
+      - -DGDCM_INSTALL_PYTHONMODULE_DIR=/app/lib/python3.8/site-packages/
+      - -DGDCM_USE_VTK=OFF
+      - -DGDCM_BUILD_DOCBOOK_MANPAGES:BOOL=OFF
+    sources:
+      - type: archive
+        url: https://github.com/malaterre/GDCM/archive/v3.0.10.tar.gz
+        sha256: a3fd3579ca0bb4a2a41ee18770e7303b22fd5460c3a2000e51ff0be6799e1d85
+        x-checker-data:
+          type: anitya
+          project-id: 883
+          stable-only: true
+          url-template: https://github.com/malaterre/GDCM/archive/v$version.tar.gz
+    cleanup:
+      - /bin
+
+  - name: Eigen3
+    buildsystem: cmake-ninja
+    builddir: true
+    sources:
+      - type: archive
+        url: https://gitlab.com/libeigen/eigen/-/archive/3.4.0/eigen-3.4.0.tar.bz2
+        sha256: b4c198460eba6f28d34894e3a5710998818515104d6e74e5cc331ce31e46e626
+        x-checker-data:
+          type: anitya
+          project-id: 13751
+          stable-only: true
+          url-template: https://gitlab.com/libeigen/eigen/-/archive/$version/eigen-$version.tar.bz2
+    cleanup:
+      - '*'
+
+  - name: ITK
+    buildsystem: cmake
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE:STRING=Release
+      - -DITK_LEGACY_REMOVE:BOOL=ON
+      - -DITK_LEGACY_SILENT:BOOL=ON
+      - -DModule_ITKBridgeNumPy:BOOL=ON
+      - -DBUILD_TESTING:BOOL=OFF
+      - -DITK_WRAP_unsigned_short:BOOL=ON
+      - -DITK_WRAP_rgb_unsigned_char:BOOL=OFF
+      - -DITK_WRAP_rgba_unsigned_char:BOOL=OFF
+      - -DITK_BUILD_DEFAULT_MODULES:BOOL=OFF
+      - -DITKGroup_Core:BOOL=ON
+      - -DITKGroup_Filtering:BOOL=ON
+      - -DITKGroup_Segmentation:BOOL=ON
+      - -DITKGroup_Nonunit:BOOL=ON
+      - -DPython_ADDITIONAL_VERSIONS:STRING=3
+      - -DITK_WRAP_PYTHON:BOOL=ON
+      - -DBUILD_EXAMPLES:BOOL=OFF
+      - -DBUILD_SHARED_LIBS:BOOL=ON
+      - -DITK_USE_SYSTEM_LIBRARIES:BOOL=ON
+      - -DITK_USE_SYSTEM_CASTXML:BOOL=ON
+      - -DITK_USE_SYSTEM_GDCM:BOOL=ON
+      - -DITK_USE_SYSTEM_SWIG:BOOL=ON
+      - -DITK_USE_SYSTEM_VXL:BOOL=OFF
+      - -DITK_FORBID_DOWNLOADS=ON
+    sources:
+      - type: archive
+        url: https://github.com/InsightSoftwareConsortium/ITK/archive/refs/tags/v5.2.1.tar.gz
+        sha256: 6022b2b64624b8bcec3333fe48d5f74ff6ebceb3bdf98258ba7d7fbbc76b99ab
+        x-checker-data:
+          type: anitya
+          project-id: 223473
+          stable-only: true
+          url-template: https://github.com/InsightSoftwareConsortium/ITK/archive/refs/tags/v$version.tar.gz
+
+  - name: alizams
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_BUILD_TYPE:STRING=Release
+      - -DCMAKE_INSTALL_PREFIX=/app
+      - -DCMAKE_INSTALL_LIBDIR=/app/lib
```

```suggestion
      - -DCMAKE_INSTALL_PREFIX=/app
      - -DCMAKE_INSTALL_LIBDIR=/app/lib
```

You shouldn't need these, Flatpak-builder handle this automatically.